install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -356,15 +357,14 @@ claim.
 
 [[nfs-automation]]
 == Automation
-As discussed, clusters can be provisioned with persistent storage using NFS in
-the following way:
-
-- Disk partitions can be used to link:#nfs-enforcing-disk-quotas[enforce storage
-quotas].
-- Security can be enforced by link:#nfs-volume-security[restricting volumes] to
-the project that has a claim to them.
-- link:#nfs-reclaiming-resources[Reclamation of discarded resources] can be
-configured for each persistent volume.
+Clusters can be provisioned with persistent storage using NFS in the following
+ways:
+
+- link:#nfs-enforcing-disk-quotas[Enforce storage quotas] using disk partitions.
+- Enforce security by link:#nfs-volume-security[restricting volumes] to the
+project that has a claim to them.
+- Configure link:#nfs-reclaiming-resources[reclamation of discarded resources] for
+each PV.
```

Comment:
Under "Group IDs", in the pod fragment example, `runAsUser` is comment out and with a comment stating "65534 from above has been commented out here". At some point in my changes I removed this line since I moved group ID before user ids. (Previously, when user ids came before group ids this comment made more sense.) I think the `runAsUser` stanza should be removed.